packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -1,14 +1,89 @@
-import { Text } from "@stenajs-webui/core";
+import { Indent, Row, Text } from "@stenajs-webui/core";
+import {
+  Icon,
+  stenaExclamationCircle,
+  stenaExclamationTriangle,
+  stenaInfoCircle,
+} from "@stenajs-webui/elements";
+import TippyComponent, {
+  TippyProps as TippyComponentProps,
+} from "@tippyjs/react";
+import cx from "classnames";
 import * as React from "react";
-import { Popover, PopoverProps, PopoverVariant } from "../popover/Popover";
+import { TippyCallbackRef } from "../../hooks/UseTippyInstance";
+import styles from "./Tooltip.module.css";
 
-export interface TooltipProps extends Partial<Omit<PopoverProps, "content">> {
-  variant?: PopoverVariant;
+type TooltipVariant = "info" | "warning" | "error";
+
+interface TippyProps
+  extends Partial<Omit<TippyComponentProps, "theme" | "render">> {
+  tippyRef?: TippyCallbackRef<HTMLDivElement>;
+}
+
+export interface TooltipProps extends TippyProps {
+  variant?: TooltipVariant;
   label: string;
+  multiRow?: boolean;
 }
 
-export const Tooltip: React.FC<TooltipProps> = ({ label, ...popoverProps }) => {
+const variantIcons = {
+  info: stenaInfoCircle,
+  warning: stenaExclamationCircle,
+  error: stenaExclamationTriangle,
+};
+
+const textComponent = (label: string, maxWidth: number) => (
+  <Indent spacing={0.5} display={"inline-block"}>
+    <Text
+      color="white"
+      size={"small"}
+      variant="bold"
+      style={{ display: "flex", maxWidth: maxWidth }}
+    >
+      {label}
+    </Text>
+  </Indent>
+);
+
+export const Tooltip: React.FC<TooltipProps> = ({
+  visible,
+  trigger = "mouseenter",
+  children,
+  tippyRef,
+  delay = 0,
+  variant,
+  content,
+  multiRow,
+  arrow = true,
+  label,
+  ...tippyProps
+}) => {
   return (
-    <Popover {...popoverProps} content={<Text size={"small"}>{label}</Text>} />
+    <TippyComponent
+      interactive
+      className={styles.noPadding}
+      trigger={visible !== undefined ? undefined : trigger}
+      visible={visible}
+      theme={"dark"}
```

Comment:
Tippy's dark theme won't do it. That's `#333` for background. We want ui-800.

Could we possibly use `Popover` like before and make a dark theme? I see no reason for duplicating the `TippyComponent` code.

Reply:
The Popover component doesn't support icons per default and last time I modified it, it was rejected due to changing the logic of it.

I'll fix your suggested changes and then we'll see how it looks. :)

Reply:
🤷‍♂️ What do the rest of the team say, should we just reject this and change the popover component instead? @nikteg  @mattias800  @juiceit